### PR TITLE
Fix fonts for OSS

### PR DESCRIPTION
**Pull request by orta**
When debugging with @mennenia earlier we spotted that the fonts we're being set up in the prefix header correctly.

### CODE REVIEW

File: Classes/Categories/Categories.h [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
-#if __has_include(<Artsy + UIFonts/UIFont + ArtsyFonts.h>)
+#if __has_include(<Artsy+UIFonts/UIFont+ArtsyFonts.h>)
 #import <Artsy+UIFonts/UIFont+ArtsyFonts.h>
 #endif
 
-#if __has_include(<Artsy + OSSUIFonts/UIFont + OSSArtsyFonts.h>)
+#if __has_include(<Artsy+OSSUIFonts/UIFont+OSSArtsyFonts.h>)
 #import <Artsy+OSSUIFonts/UIFont+OSSArtsyFonts.h>
 #endif
```
Comment (orta) [inline Classes/Categories/Categories.h]: this file is all that really matters

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Error
    ------------- | ------------
        :no_entry_sign: | No CHANGELOG changes made
    

  

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by mennenia** [comment]
It's workiiing :D. Build succeeded ✅

(you obviously all know this, I just wanted to test it lol)

**Comment by sarahscott** [comment]
wooohooo

**Pull request merged**